Multilingual.php

```diff
@@ -754,5 +808,15 @@ public function exportData($pid, $lang){
 
 		echo $data;
 	}
+	
+	public function denumerateElement($element_enum) {
+		$pieces = explode("\\n", $element_enum);
+		$elements = [];
+		foreach($pieces as $piece) {
+			list($a, $b) = explode(',', $piece, 2);
+			$elements[trim($a)] = trim($b);
+		}
+		return $elements;
+	}

```

Comment:
Has this function been pretty well tested?  If not, it looks like it'd be really easy to write a unit test for.

Reply:
Method removed -- method obviated by use of json_decode.